cmake/compile_and_update_tests.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# This script can modify the test results in the source
+# directory to the official tester results, if docker is installed
+# on the system. Start this script by running the following command
+# after replacing ASPECT_SOURCE_DIR with the directory of your ASPECT
+# folder:
+#
+# docker run -v ASPECT_SOURCE_DIR:/home/dealii/aspect --name=aspect-tester --rm -it dealii/dealii:v8.5.0-gcc-mpi-fulldepscandi-debugrelease bash /home/dealii/aspect/cmake/compile_and_update_tests.sh
+
+# This command executes the following shell script *inside* the docker container
+# that contains the official ASPECT test system. Note that by mounting your
+# ASPECT folder into the container you are actually changing reference test
+# results on the host system (i.e. your computer).
+
+echo "Starting official tester and compiling source ..."
+
+SRC_PATH=`dirname $0`
+SRC_PATH=`cd $SRC_PATH/..;pwd`
+
+mkdir -p ${SRC_PATH}/tester-build
+cd ${SRC_PATH}/tester-build
```

Comment:
you are putting the build output and temporary files in the mounted folder, meaning they will be left around after running the container, right? This mean rerunning the compilation in the container can be fast, but you are not getting a clean build...

Reply:
True, not sure what is better. Personally, I often rerun the tester multiple times fixing a bug or a test at a time and a clean compilation every time would take forever. So the current state is how I would use it, but I am not sure if this is something everybody would do. Alternatively, we could introduce an environment variable like `ASPECT_TESTER_CLEAN_BUILD` that can be set when running the docker container to force a clean build.

Reply:
If this works reliably for you, then we should leave it as is.